torch/_decomp/decompositions.py

```diff
@@ -2763,6 +2763,11 @@ def _reshape_alias(x, shape, *args):
     return aten.view(x, shape)
 
 
+@register_decomposition(aten._unsafe_permute)
+def _unsafe_permute(x, dims):
+    return aten.permute.default(x, dims)
```

Comment:
hmmmm shouldn't inductor be opted into using this decomp (in its decomp [list](https://github.com/pytorch/pytorch/blob/3ace14eb8b5e437322acf962d2f170561fd4e3bc/torch/_inductor/decomposition.py#L18))? Then you won't need to bother telling e.g. the partitioner about it.